### ISSUE TITLE
86Box Can't Recapture Mouse&Keyboard after new ISO is Loaded into CDROM

### ISSUE THREAD

**Issue by Mast3rBlast3r** [86Box/86Box#1921]
**Describe the bug**

I noticed something that happened many times, always at the same critical moment.. when I tried from the Menu to change ISO file in the virtual CDROM device, after the mouse is released from capturing with middle click.. the 86Box cannot recapture the mouse and keyboard by clicking in the 86Box screen again, only Hard Reset remains..

then I tested the same configuration with PCem17, and I got a different result.. the mouse continued to work, only the keyboard was lost, I know because the game asked for a serial number and I couldn't type, but after resetting Render Driver mode from the Menu I regained the keyboard, I was able to continue.. 

**To Reproduce**

launch your 86Box with Windows 95 or Windows 98 and load from the Menu some ISO file into CDROM device, then click with mouse into 86Box screen again.. 

**Expected behavior**

stronger recapture code, it's not polished, maybe we need some icon on Status bar as indicator to show is keyboard captured or not, like in VirtualBox? I have Windows 95 and Windows 98 installed in almost latest VirtualBox, VirtualBox will capture mouse and keyboard immediately after launch, and release it from capturing only after CTRL taster is pressed, and always works..

**Screenshots**
I have one moment after freezing with Windows 95, and two with Windows 98 after game installer is launched.. and one EXTRA with VirtualBox 6.0.12 with Windows 95, DirectX and Direct3D are working, but is all kind of slow for serious gameplay..

**Desktop:**

 - OS: Windows 10
 - 86Box version: v3.00 build x32-b3336
 - Build information: ASUS P/I-P55T2P4 with AMD K6 (Model 6) 233 CPU, Memory 256 MB, 3dfx Voodoo3 3000, Standard PS/2 Mouse, Sound Blaster 16 [220, 7,1,5]; System MIDI, Network: none, Ports: Default, Storage Controllers: Internal, Internal, HDD 4GB, Floppy 3.5 1.44M, CD-ROM ATAPI 24x  

**Additional context**

a temporary solution for this problem is to change ISO file in CDROM device immediately after booting, before Windows boots, I tested this with Windows 95 and Windows 98SE installed, I made a clean installation without errors, I have a clear view in Device Manager, no conflicts or missing drivers, so please fix it, and thank you..

![86Box Capture0](https://user-images.githubusercontent.com/95438261/145222751-affb3c1e-5a50-4314-a6a8-2760fd047d9b.png)
![86Box Capture1](https://user-images.githubusercontent.com/95438261/145222759-03a7bc18-b3d4-4148-b41b-3547bf42f104.png)
![86Box Capture2](https://user-images.githubusercontent.com/95438261/145222782-373b68b0-2f3b-4d49-b7ab-28c3d105080f.png)
![86Box Capture3](https://user-images.githubusercontent.com/95438261/145222789-931a0508-be84-4be3-84ee-833618f853df.png)
![VBox win95 capture](https://user-images.githubusercontent.com/95438261/145222798-71f4d1fd-1e55-476d-bfc0-96672d2b2d88.png)

**Comment by Dungeonseeker** [comment]
Keyboard input is done automatically unless you have the "File > Keyboard needs capture" option ticked.

I have had this a few times with mouse capture though.

**Comment by OBattler** [comment]
I have no idea what causes this since I can't reproduce it at all.

**Comment by CrazyCSIW6** [comment]
Same thing here.

**Comment by OBattler** [comment]
Closing this as a duplicate of #481.
